duet/duet.model.lkml

```diff
@@ -4,6 +4,18 @@ include: "views/*"
 include: "explores/*"
 include: "dashboards/*"
 
+##########################################################################################
+# below are views saved into the shong-dev directory specifically for the MR1 dashboard
+# do not use those views for other projects, as they are specific to the MR1 dashboard and
+# haven't been developed for general consumption
+include: "/shong-dev/views/version_uplift.view.lkml"
```

Comment:
Let's move these to the duet folder, e.g.: `/duet/mr1/views/version_uplift.view.lkml`. That will be a bit easier to understand, and will help prevent a clutter of dirs in the production repo.

Reply:
Moving further, you could move all the explores to an `/duet/mr1/explores.explore.lkml` file, and import just that in the model: `include: "/duet/mr1/explores/explore/lkml`. (this is optional).

Reply:
reorganized explores and directories